package.json

```diff
@@ -204,6 +205,8 @@
     "react-dom": "17.0.2",
     "react-dropzone": "^11.4.2",
     "react-head": "3.0.2",
+    "react-instantsearch-core": "^6.18.0",
+    "react-instantsearch-dom": "^6.18.0",
```

Comment:
I mean — I would not have used this library and would much rather us write our own code. Though there's the question of whether or not we have the time to do this?

![](https://static.damonzucconi.com/_capture/Htchh4g4.png)
![](https://static.damonzucconi.com/_capture/DFoeEnDi.png)

This is a lot of code to add to nearly every major page on the site.

cc @icirellik

Reply:
> I mean — I would not have used this library and would much rather us write our own code. Though there's the question of whether or not we have the time to do this?

Good question. I'll see what can be done and if there is any way to reduce the size

Reply:
Made some changes, now `react-instantsearch-dom` is not used

Reply:
@dzucconi Can I mark this thread as resolved?

Reply:
👍